app/views/fandoms/unassigned.html.erb

```diff
@@ -3,9 +3,18 @@
 <!--/descriptions-->
 
 <!--subnav-->
-<%= will_paginate(@fandoms) %>
+<% # Filters button for narrow screens jumps to filters when JavaScript is disabled and opens filters when JavaScript is enabled %>
+<ul class="navigation actions" role="navigation">
+  <li class="narrow-shown hidden">
+    <a href="#fandom-filters" id="go_to_filters">
+      <%= ts("Filters") %>
```

Comment:
I don't think so. We haven't discussed how exactly we're going to do it (e.g. naming conventions for keys), and any time we try to use it, the pull request ends up being delayed for an eternity. So we've been sticking with the current system.